File: easy/3512. Minimum Operations to Make Array Sum Divisible by K/main.java
// 3512. Minimum Operations to Make Array Sum Divisible by K
class Solution {
    public int minOperations(int[] nums, int k) {
        long sum = 0;
        for (int num : nums) {
            sum += num;
        }
        
        int remainder = (int)(sum % k);
        return remainder;
    }
}

File: easy/3512. Minimum Operations to Make Array Sum Divisible by K/main.rs
// 3512. Minimum Operations to Make Array Sum Divisible by K
impl Solution {
    pub fn min_operations(nums: Vec<i32>, k: i32) -> i32 {
        let sum: i64 = nums.iter().map(|&x| x as i64).sum();
        (sum % k as i64) as i32
    }
}